react-graph/src/PointNode.tsx

```diff
@@ -205,6 +214,15 @@ export class PointNode {
         return children;
     }
 
+    private trimTree(): void {
+        const fullData = this.GetFullData();
+        if (fullData.length < MaxTotalPoints) return
```

Comment:
don't we have a `Count` property? I don't think we need to go into `GetFullData` for this